topic_modeling/train_models/train_models.py

```diff
@@ -0,0 +1,141 @@
+import os
+import glob
+import re
+import nltk
+import string
+import pickle
+from sklearn.feature_extraction.text import CountVectorizer
+from sklearn.decomposition import LatentDirichletAllocation
+from sklearn.feature_extraction import text
+from nltk.tokenize import word_tokenize
+from nltk.stem.porter import PorterStemmer
+from nltk.corpus import stopwords
+from nltk.stem.wordnet import WordNetLemmatizer
+
+nltk.download("stopwords")
+nltk.download("wordnet")
+
+stop = set(stopwords.words('english'))
+exclude = set(string.punctuation)
+lemma = WordNetLemmatizer()
```

Comment:
even if it is to be used as a script, and no line would be reused multiple times, I would recommend to modularize it into function calls, and then use that `if __name__ == '__main__`:` condition to run your "main" function for the script.  This 
- would allow to (unit)test pieces of the script if desired
- might improve readability since you would not have one long flow of code, but some high level invocation and then details implemented by any particular function
- could help catch bugs easier where you might be using something what was defined somewhere long ago in irrelevant code etc

Reply:
made some moves toward that! could probably modularize even further.